docs/specs/sections/definitions.tex

```diff
@@ -0,0 +1,98 @@
+\subsection{The Roles Involved} 
+
+\begin{itemize}
+    \item \textbf{User:} an entity that interacts with the wallet to request licenses and prove ownership of those.
+    \item \textbf{License Provider (LP):} an entity that receives requests for licenses, and upon acceptance, issues them.
+    \item \textbf{Service Provider (SP):} the entity that provides a service upon verification that a service request is correct. The SP may be the same as the LP entity or a different one.
+\end{itemize}
+
+\subsection{The Elements Involved} 
+
+\begin{itemize}
+    \item \textbf{Request:} a request includes the encryption of a stealth address belonging to the user, where the license has to be sent to, and a symmetric key. The structure is as follows:
+
+    \begin{center}
+        \begin{tabular}{ | p{0.15\linewidth} | p{0.2\linewidth} | p{0.55\linewidth} | } 
+        \hline
+        \textbf{Element} & \textbf{Type} & \textbf{Info.} \\
+        \hline
+        $(\rpk, R_{\req})$ & StealthAddress & It is a request stealth address of the LP. \\
+        $\enc$ & PoseidonCipher[6] & It is the encryption of a license stealth address of the user and a symmetric key. \\
+        $\nonce$ & BlsScalar & Randomness needed to compute $\enc$. \\ 
+        \hline
+        \end{tabular}
+    \end{center}
+
+    \item \textbf{License:} a license is an asset that represents the right of a user to use a given service. The structure is as follows:
+
+    \begin{center}
+        \begin{tabular}{ | p{0.15\linewidth} | p{0.2\linewidth} | p{0.55\linewidth} | } 
+        \hline
+        \textbf{Element} & \textbf{Type} & \textbf{Info.} \\
+        \hline
+        $(\lpk, R_{\lic})$ & StealthAddress & It is a license stealth address of the user. \\
+        $\enc$ & PoseidonCipher[4] & It is the encryption of some user attributes and its signature. \\
```

Comment:
By "its signature" do you mean the user's signature? If so, it should be _It is the encryption of some user attributes and ~~its~~ **their** signature".

Reply:
no, the signature of the attributes